### PR TITLE
Add ansible_uptime_seconds support for AIX

### PR DESCRIPTION
**Pull request by schroeert**
##### SUMMARY

AIX was missing the ansible_uptime_seconds fact, which is available for Linux.
I implemented get_uptime_facts for AIX.

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

The ansible.builtin.setup module collects various facts, such as ansible_uptime_seconds. This fact is implemented for most operating systems, with the exception of AIX.

### CODE REVIEW

**Review by neikei — APPROVED** [review]
/lgtm - Many thanks for pushing content for AIX 🚀

File: lib/ansible/module_utils/facts/hardware/aix.py [hunk 1]
```diff
@@ -123,6 +125,26 @@ def get_memory_facts(self):
...
+            days = int(data[0])
+            hours = int(data[1])
+            minutes = int(data[2])
+            seconds = int(data[3])
+            # Calculate uptime in seconds
+            uptime_seconds = (days * 86400) + (hours * 3600) + (minutes * 60) + seconds
+            uptime_facts['uptime_seconds'] = int(uptime_seconds)
+
+        return uptime_facts
+
```
Comment (Akasurde) [inline lib/ansible/module_utils/facts/hardware/aix.py]:
```suggestion
    def get_uptime_facts(self):
        uptime_facts = {}
        # On AIX, there are no options to get the uptime directly in seconds.
        # Your options are to parse the output of "who", "uptime", or "ps".
        # Only "ps" always provides a field with seconds.
        ps_bin = self.module.get_bin_path("ps")
        if ps_bin is None:
            return uptime_facts
        
        ps_cmd = [ps_bin, "-p", "1", "-o", "etime="]
        
        rc, out, err = self.module.run_command(ps_cmd)
        if rc != 0:
            return uptime_facts

        # Parse out
        if out:
            lines = out.splitlines()
            data = lines[0].replace(':', '-').split('-')
            try:
                days = int(data[0])
                hours = int(data[1])
                minutes = int(data[2])
                seconds = int(data[3])
            except (IndexError, ValueError):
                return uptime_facts
            # Calculate uptime in seconds
            uptime_seconds = (days * 86400) + (hours * 3600) + (minutes * 60) + seconds
            uptime_facts['uptime_seconds'] = int(uptime_seconds)

        return uptime_facts

```

**Review by Akasurde — CHANGES_REQUESTED** [review]
Add an unit test for this change. 
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

**Comment by schroeert** [comment]
Added the changelog_fragment and tested the provided changes on AIX. I'm not sure why the checks failed. There seems to be some issue with the Pipeline.
```
00:36 WARNING: 400: invalid: Azure Pipelines job ansible/ansible:128783.1:Remote.test_macos_14_3_1.__default public key 'ansible-core-ci/public-key.pem' not found.

**Comment by webknjaz** [comment]
Restarted the failing jobs.

**Review by Akasurde — APPROVED** [review]

**Pull request merged**